packages/components/src/border-control/border-control-dropdown/component.tsx

```diff
@@ -224,22 +222,20 @@ const BorderControlDropdown = (
 						/>
 					) }
 				</VStack>
-			</DropdownContentWrapper>
-			{ showResetButton && (
-				<DropdownContentWrapper paddingSize="none">
+				<div className={ resetButtonClassName }>
 					<Button
-						className={ resetButtonClassName }
 						variant="tertiary"
 						onClick={ () => {
 							onReset();
-							onClose();
 						} }
+						disabled={ ! showResetButton }
```

Comment:
Let's rename this variable to something closer to its purpose, ie. `enableResetButton`